### DISCUSSION TITLE
PSfalcon Put-And-Run

### DISCUSSION THREAD

**Discussion by Xlichtx011** [CrowdStrike/psfalcon#324] [Q&A]
I am working on a small project hehe.

I tried the following  commands on Psfalcon and RTR
Run C:/Tools/test.exe  > result as it only run in the background no window appear
Put-and-Run test.exe >  result as it only run in the background no window appear
Tried to launch another exe that was already on the host >  result as it only run in the background no window appear

am i doing something wrong?

**Comment by bk-cs** [comment]
No, this is all expected behavior. RTR runs as SYSTEM and will always run in the "background" (i.e. not the currently logged in user).
